doc/changelog/08-vernac-commands-and-options/18537-fix-search-scheme.rst

```diff
@@ -0,0 +1,8 @@
+- **Fixed:**
+  :cmd:`Search` with modified `is:Scheme` restricted the search to inductives
+  which have schemes instead of the schemes themselves.
+  For instance `Search nat is:Scheme` with just the prelude loaded would return `le`
+  i.e. the only inductive whose type mentions `nat`
```

Comment:
```suggestion
  :cmd:`Search` with modifier `is:Scheme` restricted the search to inductive types
  which have schemes instead of the schemes themselves.
  For instance `Search nat is:Scheme` with just the prelude loaded would return `le`
  i.e. the only inductive type whose type mentions `nat`
```

Reply:
Maybe you could finish your example and state what it does now?